Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -0,0 +1,180 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+extension JNISwift2JavaGenerator {
+  func writeExportedJavaSources() throws {
+    var printer = CodePrinter()
+    try writeExportedJavaSources(&printer)
+  }
+
+  package func writeExportedJavaSources(_ printer: inout CodePrinter) throws {
+    for (_, ty) in analysis.importedTypes.sorted(by: { (lhs, rhs) in lhs.key < rhs.key }) {
+      let filename = "\(ty.swiftNominal.name).java"
+      logger.info("Printing contents: \(filename)")
+      printImportedNominal(&printer, ty)
+
+      if let outputFile = try printer.writeContents(
+        outputDirectory: javaOutputDirectory,
+        javaPackagePath: javaPackagePath,
+        filename: filename
+      ) {
+        print("[swift-java] Generated: \(ty.swiftNominal.name.bold).java (at \(outputFile))")
```

Comment:
this can use the `logger`
```suggestion
        logger.info("[swift-java] Generated: \(ty.swiftNominal.name.bold).java (at \(outputFile))")
```